pkgs/top-level/all-packages.nix

```diff
@@ -20215,7 +20215,8 @@ with pkgs;
 
   pari = callPackage ../applications/science/math/pari { tex = texlive.combined.scheme-basic; };
   gp2c = callPackage ../applications/science/math/pari/gp2c.nix { };
-  pari-unstable = callPackage ../applications/science/math/pari/unstable.nix {};
```

Comment:
Why did you remove pari-unstable?

Reply:
I removed it because `pari-unstable` is older than regular `pari` and not updated anymore (http://pari.math.u-bordeaux.fr/pub/pari/unstable/).

But that wasn't meant to be in this PR, I'll remove it.